assets/base.css

```diff
@@ -1762,20 +1844,34 @@ input[type='checkbox'] {
 
 /* component-quantity */
 .quantity {
-  border: var(--inputs-border-width) solid rgba(var(--color-foreground), var(--inputs-border-opacity));
-  background: rgb(var(--color-background));
   color: rgba(var(--color-foreground));
-  border-radius: var(--inputs-radius);
   position: relative;
-  height: 4.5rem;
-  width: calc(14rem / var(--font-body-scale));
+  width: calc(14rem / var(--font-body-scale) + var(--inputs-border-width) * 2);
   display: flex;
-  filter: drop-shadow(
-    var(--inputs-shadow-horizontal-offset)
-    var(--inputs-shadow-vertical-offset)
-    var(--inputs-shadow-blur-radius)
-    rgba(var(--color-shadow), var(--inputs-shadow-opacity))
-  );
+  border-radius: var(--inputs-radius);
+  min-height: calc((var(--inputs-border-width) * 2) + 4.5rem);
+}
+
+.quantity:after {
```

Comment:
I was thinking the same: 
![](https://screenshot.click/22-01-o2c76-fb9u8.png)

Reply:
Both should be fixed now 👍 https://screenshot.click/13-08-4o75n-jilaq.png

But something I noticed is that by adding this fix, we get the focus on click as well since the element is in focus: https://screenshot.click/13-14-e2lms-dn808.mp4 🤔

Reply:
You may need something like this as well
```
.newsletter-form__button:not(:focus-visible) {
  box-shadow: inherit;
}
```

Reply:
For the input to have the same radius, it might be tricky for some elements I think 🤔  but it can be something to look into later.

Reply:
We need to shift our focus outline approach to fully use `box-shadow` instead of `outline` everywhere to fully get radius outlines working in safari regardless (safari's outline doesn't respect radius). So we probably don't want to spend too much time worrying about that kind of detail.